packages/account-template/src/accountLinks.ts

```diff
@@ -0,0 +1,97 @@
+import type { CeramicApi, CeramicDocument } from './types'
+
+import { createLink } from '3id-blockchain-utils'
+import { AccountID } from 'caip'
+
+class AccountLinks {
+  private _accountLinkDocuments: Record<string, CeramicDocument>;
+
+  constructor (public ceramicDoc: CeramicDocument, private _ceramic: CeramicApi) {
+    this._accountLinkDocuments = {}
+    this.ceramicDoc.on('change', this._loadAccountLinkDocs.bind(this))
+  }
+
+  list (): Array<{ account: AccountID; docId: string }> {
+    return Object.entries(this._accountLinkDocuments).map(([address, doc]) => {
+      return {
+        account: new AccountID(address),
+        docId: doc.id
+      }
+    })
+  }
+
+  async add (account: AccountID | string, opts: { proof?: Record<string, any>; provider?: any }): Promise<void> {
+    if (typeof account === 'string') {
+      let [address, chainId] = account.split('@') // eslint-disable-line prefer-const
+      if (!chainId) {
+        chainId = 'eip155:1'
+      }
+      account = new AccountID({ address, chainId })
+    }
```

Comment:
We should assume that if this a string it's a caip-10 accountId. If so we can just do `new AccountID(<string>)`. If it's not we should just throw an error.